releases/compiler/parallel_test.go

```diff
@@ -28,23 +30,38 @@ func TestParallel_Compile(t *testing.T) {
 			Result:      &binary1,
 			Name:        "binary1",
 			Target:      "../..",
-			Source:      "./testing/compiler/internal/cmd",
```

Comment:
This was confusingly pointing at the wrong directory

Reply:
Copy and past issue i assume 😅